libsolidity/interface/CompilerStack.cpp

```diff
@@ -215,6 +216,8 @@ bool CompilerStack::analyze()
 				{
 					contract->setDevDocumentation(Natspec::devDocumentation(*contract));
 					contract->setUserDocumentation(Natspec::userDocumentation(*contract));
```

Comment:
I don't know the first thing about documentation, so I am not qualified to answer your question.